sessions/forecasting-models.qmd

```diff
@@ -232,7 +232,7 @@ We will now use the mechanistic and statistical models to forecast the number of
 source(here::here("snippets", "simulate-onsets.r"))
 onset_df
 
-cutoff <- 71
+cutoff <- 41
```

Comment:
intended?

Reply:
the run time was annoying me (because its 71 + 28) so I changed. I think it being quick is worth the inconsistency